lib/linguist/heuristics.yml

```diff
@@ -712,7 +712,7 @@ disambiguations:
 - extensions: ['.script']
   rules:
   - language: URScript
-    pattern: '^\s*def\s+[a-zA-Z_]\w*\s*\([^)]*\)\s*:[\s\S]*?\send($|\s)'
+    pattern: '(^\s*|\s+)def\s+[a-zA-Z_]\w*\s*\([^)]*\)\s*:[\s\S]*?\send($|\s)'
```

Comment:
This is unnecessary (`\s*` is zero or more which includes `\s+` which is one or more) and introduces a ReDoS.

Reply:
I see. I reverted the commit.